pyproject.toml

```diff
@@ -13,65 +13,65 @@ dependencies = [
     "Click~=8.1.3",
     "filelock~=3.8.0",
     "filetype~=1.2.0",
-    "GitPython~=3.1.30",
+    "GitPython~=3.1.31",
     "Jinja2~=3.1.2",
-    "Pillow~=9.3.0",
-    "PyJWT~=2.4.0",
+    "Pillow~=9.5.0",
+    "PyJWT~=2.7.0",
     "PyMySQL==1.0.3",
-    "PyPDF2~=2.1.0",
+    "pypdf~=3.9.1",
     "PyPika~=0.48.9",
     "PyQRCode~=1.2.1",
     "PyYAML~=6.0",
     "RestrictedPython~=6.0",
-    "WeasyPrint==52.5",
-    "Werkzeug~=2.2.2",
+    "WeasyPrint==59.0",
```

Comment:
We can drop weasyprint? No one is supposed to be using PFB Beta.

Reply:
Kept it because:

https://github.com/frappe/frappe/issues/21092